src/watchers/indexable-meta.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * WordPress Post Meta watcher.
+ *
+ * @package Yoast\YoastSEO\Watchers
+ */
+
+namespace Yoast\YoastSEO\Watchers;
+
+use Yoast\YoastSEO\Exceptions\No_Indexable_Found;
+use Yoast\YoastSEO\Models\Indexable_Meta as Indexable_Meta_Model;
+use Yoast\YoastSEO\Yoast_Model;
+
+/**
+ * Fills the Indexable Meta according to Post data.
+ */
+class Indexable_Meta {
```

Comment:
I like the functionality of this class but it's not a watcher.

I would implement these function on the `Indexable` model instead.